ipn/ipnlocal/local.go

```diff
@@ -4891,7 +4902,7 @@ func (b *LocalBackend) setAutoExitNodeIDLockedOnEntry(unlock unlockOnce) {
 		return
 	}
 	prefsClone := prefs.AsStruct()
-	newSuggestion, err := b.suggestExitNodeLocked()
+	newSuggestion, err := b.suggestExitNodeLocked(b.netMap)
```

Comment:
Perhaps prefer to use `nil` as the netmap parameter when it's `b.netMap` to be consistent with the documented way to use it? Otherwise we could just have `suggestExitNodeLocked` skip the `nil` check and say it always needs to be filled in.